WcaOnRails/spec/factories/persons.rb

```diff
@@ -9,7 +9,7 @@
       id
     end
     subId { 1 }
-    name { Faker::Name.name }
+    name { Faker::Name.name.gsub(" DVM", "") } # DVM removed to prevent unwanted warnings (we don't allow titles as suffixes)
```

Comment:
See if you can merge this into a helper method to use in both factories.

Reply:
I actually have an alternate - arguably cleaner if it works - suggestion: Faker generates the names based on a locale file, if I'm not mistaken we use `:en` and therefore I think we could add this:
```yml
en:
  faker:
      prefix: []
      suffix: []
```
into a `faker.en.yml` in [this folder](https://github.com/thewca/worldcubeassociation.org/tree/master/WcaOnRails/config/locales), similarly to how we override defaults for devise/carrierwave.
With this config I believe Faker should no longer generate names with either prefixes or suffixes (which I think is the default we want, but maybe I'm missing something).

Reply:
I love this! That's the kind of solution I was hoping for, but didn't know it existed. Thanks!
Timo, please proceed like Philippe suggested.

Reply:
I added the yml file in config/locales: doesn't work correctly. I'm lost on where faker is actually configured; is there another location where it could be possible to add the commands to override the defaults?

Reply:
I just tried it out locally and it does work, however I missed a nesting level (`name:`) in my original suggestion, sorry for that.
Upon trying it out I also noticed it would generate names like ` John Doe` or `John Doe ` (with an extra space at the beginning or at the end), which is far from ideal.
However a still as simple solution is to override how the name is generated:
```yml
en:
  faker:
    name:
      name:
        - "#{first_name} #{last_name}"
```
The double `name` is intended, see the original faker configuration [here](https://github.com/faker-ruby/faker/blob/main/lib/locales/en/name.yml).

Reply:
Ah, glancing at the original configuration helps a ton! Removing all suffixes wouldn't be ideal for validators since we allow at least Jr., Sr. and numerals. I cut down the suffix list to those, and removed prefixes:

`en:
  faker:
    name:
      suffix: [ Jr., Sr., I, II, III, IV, V ]
      name:
        - "#{first_name} #{last_name} #{suffix}"
        - "#{first_name} #{last_name}"
        - "#{first_name} #{last_name}"
        - "#{first_name} #{last_name}"
        - "#{first_name} #{last_name}"
      name_with_middle:
        - "#{first_name} #{last_name} #{last_name} #{suffix}"
        - "#{first_name} #{last_name} #{last_name}"
        - "#{first_name} #{last_name} #{last_name}"
        - "#{first_name} #{last_name} #{last_name}"
        - "#{first_name} #{last_name} #{last_name}"
`

I'll commit if you agree with this!

Reply:
That sounds like a great solution :)

Reply:
I am still getting the unused key exception, what file name have you been using in your testing?

Reply:
Not Philippe but basically our code checks think that this is a general translation file and then worry because the content of the file is not actually used anywhere. This is a reasonable check to have in general, but I guess in this case we should configure it so that your file is excluded from the checks.

Reply:
Cf. `WcaOnRails/config/i18n-tasks.yml.erb`

Reply:
There is an `ignore` section towards the bottom half of the file that you can leverage.

Reply:
> I am still getting the unused key exception, what file name have you been using in your testing?

Ah I didn't actually run the tests, I just made sure the name generation did work; Gregor is right, you can simply add the faker keys to the one ignored by our [i18n tasks](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/config/i18n-tasks.yml.erb#L142).

Reply:
I added the keys to ignore_unused, I believe everything's in order now